### DISCUSSION TITLE
Flow Bite support for Angular projects

### DISCUSSION THREAD

**Discussion by dev-garcia** [themesberg/flowbite-angular#19] [Q&A]
Hi,

I am a developer from Brazil who loves working with Tailwind CSS. I use Tailwind CSS in all of my personal and professional projects because I find it flexible and easy to use.

Recently, I received an opportunity to work on a project with Angular, where the team uses Bootstrap. I particularly hate working with Bootstrap when I need to make layout changes.

That's why I'm looking for an alternative to Bootstrap. Flow Bite seems like a good option, but I have a question about the support plan.

I would like to know:

How long will Flow Bite be supported?
What are the future support plans for Flow Bite?

Thank you for your attention.

**Comment by MGREMY** [comment]
Hello there @dev-garcia

This project was "paused", because the team didn't find anyone to maintain the project.
As new maintainer has been involved in the project, I would say that the future for this repository is going to be the same as other project based on Flowbite, we'll add components from Flowbite main library as [react](https://www.flowbite-react.com/) or [svelte](https://flowbite-svelte.com/) have done.

Hoping that I answered your questions (and not too late). :)

**Reply by tomavic** [reply]
Hi @MGREMY I have came across flow bite for Angular and it make me sad that it is not ready yet like others [react](https://www.flowbite-react.com/) or [svelte](https://flowbite-svelte.com/) 


How can I contribute and maintain the library?

**Reply by MGREMY** [reply]
Hey @tomavic 

Since we have to do a deep refacto of the code, I don't know if you could help us now... I'll create a Github project, to list everything we have to do, and then when every items of the project will be done, we will add features like components, a better documentation etc...
For the moment I'm figuring out how we could refacto the better way. As I use to say, better is the base, better will be the project. If you think you could help me on that, and maybe help me to implement changes, send me a Discord message and I'll be glad to work with you 😁

**Comment by dev-garcia** [comment]
I'm using tailwind css with flowbite with angular using the standard components from the documentation and it worked fine (not bootstrap support).

**Reply by dev-garcia** [reply]
I'm from Brazil and I still don't speak English, I'm using a translator so forgive me if something sounds robotic or strange.

**Reply by MGREMY** [reply]
You're speaking of the flowbite documentation or flowbite-angular documentation ?
